app/controllers/projects/edit/steps/rezoning.js

```diff
@@ -17,4 +24,12 @@ export default class RezoningController extends Controller {
 
     return false;
   }
+
+  @action
+  setRezoningFalse() {
+    this.set('model.needRezoning', false);
+    this.set('model.needUnderlyingZoning', null);
+    this.set('model.needCommercialOverlay', null);
+    this.set('model.needSpecialDistrict', null);
+  }
```

Comment:
There are a couple of ways to handle this: 

1. [`queue` helper](https://github.com/DockYard/ember-composable-helpers#queue) in the template
This helper lets chain some actions together:
```handlebars
onclick={{action (queue (mut model.needRezoning) (mut model.needUnderlyingZoning) (mut model. needCommercialOverlay) (mut model.needSpecialDistrict)) false}}
```

2. computed property with [custom setter](https://emberjs.com/api/ember/3.5/classes/ComputedProperty)

We could have a computed property that is a custom setter which does this:
```javascript
@computed()
set creatingZoning(value) {
    this.set('model.needRezoning', value);
    this.set('model.needUnderlyingZoning', value || null);
    this.set('model.needCommercialOverlay', value || null);
    this.set('model.needSpecialDistrict', value || null);
}
```

3. Move the action to a method on the model, and call that method from your action (probably the best way)

We don't want logic for updating model state spread across different parts of the app, so we could simple add a new method to the model like this:

```javascript
// models/project.js
setRezoningFalse() {
    this.set('needRezoning', false);
    this.set('needUnderlyingZoning', null);
    this.set('needCommercialOverlay', null);
    this.set('needSpecialDistrict', null);
}
```

Then, in the action, we could have:

```javascript
  @action
  setRezoningFalse() {
  	this.get('model').setRezoningFalse()
  }
```

I think option 3 is best because it's more obvious what's happening and doesn't touch any unusual APIs.

Reply:
Went with option 3 in a5a28d1